src/dbnode/storage/index.go

```diff
@@ -466,6 +580,13 @@ func (i *nsIndex) Bootstrap(
 	return multiErr.FinalError()
 }
 
+func (i *nsIndex) BootstrapsDone() uint {
+	i.state.RLock()
+	result := i.state.bootstrapsDone
```

Comment:
Renaming this field and the method `NumBootstrapsDone()` would be more clear. I thought `bootstrapsDone` was an enum at first